src/azure-cli/azure/cli/command_modules/containerapp/commands.py

```diff
@@ -72,6 +72,12 @@ def load_command_table(self, _):
         g.custom_command('remove', 'remove_managed_identity_job', confirmation=True, supports_no_wait=True, exception_handler=ex_handler_factory())
         g.custom_show_command('show', 'show_managed_identity_job')
 
+    with self.command_group('containerapp job registry', is_preview=True) as g:
+        g.custom_command('set', 'set_registry_job', exception_handler=ex_handler_factory())
+        g.custom_show_command('show', 'show_registry_job')
+        g.custom_command('list', 'list_registry_job')
+        g.custom_command('remove', 'remove_registry_job', exception_handler=ex_handler_factory())
```

Comment:
Do you need to consider adding `confirmation=True` for the remove command?

Reply:
This command is for remove registry under containerapp, registry is part of configuration in Containerapp's property.
Consistent with other this kind of command, not need `confirmation=True`.

Reply:
OK, got it